Action.struct.h

```diff
@@ -35,11 +35,13 @@
 #include "Task.enum.h"
 #include "Trade.enum.h"
 
-// Structs.
+/** 
+  * Structs.
+  */
 struct ActionEntry {
-  unsigned char flags;        // Action flags.
-  datetime last_success;      // Time of the previous check.
-  long action_id;             // Action ID.
+  unsigned char flags;        /*!< Action flags. */
+  datetime last_success;      /*!< Time of the previous check. */
```

Comment:
Can we use previous `// Format.` without warnings, or we need to use `/* Format. */`?